src/lib/signature_kind/mina_signature_kind.mli

```diff
@@ -1,6 +1,4 @@
-type t = Mina_signature_kind_type.t =
+type t =
```

Comment:
Let's move this type to some other Ocaml package and then remove the `src/lib/signature_kind` completely.

Reply:
yes that is ultimately what will happen.